usage_examples/ner_v3_openai/README.md

```diff
@@ -0,0 +1,77 @@
+# Using GPT Models from OpenAI for Named Entity Recognition (NER)
+
+
+This example shows how you can use a model from OpenAI for Named Entity Recognition (NER).
+The NER prompt is based on the the [PromptNER](https://arxiv.org/abs/2305.15444) paper and
+utilizes Chain-of-Thought reasoning to extract named entities. It requires at least 1 example
+and does not work with zeroshot settings.
```

Comment:
We should guard against this specifically: when given an empty list as examples, the config will parse and zero scores will come out.

Reply:
Removed "It requires at least 1 example and does not work with zeroshot settings.", as we have a default example now.